Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```diff
@@ -135,6 +136,7 @@
     - id: DoorRemoteService
     - id: HoPIDCard
     - id: IDComputerCircuitboard
+    - id: ServiceTechFabCircuitboard # DeltaV - departamental lathes
```

Comment:
put these in the deltav locker fill tables

Reply:
moved real